proto/rill/runtime/v1/expressions.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package rill.runtime.v1;
+
+import "google/protobuf/struct.proto";
+
+message Expression {
+  oneof expression {
+    google.protobuf.Value value = 1;
+    string column = 2;
+    ConditionExpression condition_expression = 3;
+  }
+}
+
+message ConditionExpression {
+  Operation operation = 1;
+  repeated Expression operands = 2;
+}
+
+enum Operation {
+  OPERATION_UNSPECIFIED = 0;
+  OPERATION_EQUALS = 1;
+  OPERATION_NOT_EQUALS = 2;
+  OPERATION_LESSER = 3;
+  OPERATION_LESSER_OR_EQUALS = 4;
+  OPERATION_GREATER = 5;
+  OPERATION_GREATER_OR_EQUALS = 6;
+  OPERATION_OR = 7;
+  OPERATION_AND = 8;
+  OPERATION_BETWEEN = 9;
+  OPERATION_NOT_BETWEEN = 10;
```

Comment:
Redundant – would suggest using an "and" of two expressions instead. It's also ambiguous – i.e. are they inclusive or exclusive of the start and end values?

Reply:
These are from duckdb docs: https://duckdb.org/docs/sql/expressions/comparison_operators.html#between-and-is-not-null
I think we can convert them to compound operations in UI and keep the API simple.